src/System.Reflection.Metadata/tests/TestUtilities/LoaderUtilities.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System.IO;
+using System.Reflection.PortableExecutable;
+using Xunit;
+
+namespace System.Reflection.Metadata.Tests
+{
+    internal unsafe static class LoaderUtilities
+    {
+        public static void LoadPEAndValidate(byte[] peImage, Action<PEReader> validator)
```

Comment:
It depends. In any case, relocations are not ever applied to IL-only images (the code for it is here: https://github.com/dotnet/coreclr/blob/922cace885bfe24c18508747346e23ff7dc40d6c/src/vm/peimagelayout.cpp#L604).